tests/utils/test_verification_utils.py

```diff
@@ -28,21 +28,44 @@ def test_recover_relative_pose_from_essential_matrix(self):
         self.assertTrue(i2Ri1.equals(i2Ei1.rotation(), 1e-3))
         self.assertTrue(i2Ui1.equals(i2Ei1.direction(), 1e-3))
 
-    def test_compute_epipolar_distances(self):
-        """Test for epipolar distance computation using 2 sets of points and the essential matrix."""
-        normalized_pts_i1 = np.array([[1.0, 3.5], [-2.0, 2.0]])
-        normalized_pts_i2 = np.array([[2.0, -1.0], [1.0, 0.0]])
+    def test_recover_relative_pose_from_essential_matrix_none(self):
+        """Test for function to extract relative pose from essential matrix."""
 
-        i2Ri1 = Rot3.RzRyRx(0, np.deg2rad(30), np.deg2rad(10))
-        i2ti1 = np.array([-0.5, 2.0, 0])
-        i2Ei1 = EssentialMatrix(i2Ri1, Unit3(i2ti1))
+        # simulate correspondences and the essential matrix
+        corr_i1, corr_i2, _ = simulate_two_planes_scene(10, 10)
 
-        expected = np.array([1.637, 1.850])
+        i2Ri1, i2Ui1 = verification_utils.recover_relative_pose_from_essential_matrix(
+            None, corr_i1.coordinates, corr_i2.coordinates, Cal3Bundler(), Cal3Bundler()
+        )
 
-        computed = verification_utils.compute_epipolar_distances(normalized_pts_i1, normalized_pts_i2, i2Ei1)
+        # compare the recovered R and U with the ground truth
+        self.assertIsNone(i2Ri1)
+        self.assertIsNone(i2Ui1)
+
+    def test_compute_epipolar_distances_sed(self):
+        """Test for epipolar distance computation using 2 sets of points and the fundamental matrix."""
+        points_i1 = np.array([[1.0, 3.5], [-2.0, 2.0]])
+        points_i2 = np.array([[2.0, -1.0], [1.0, 0.0]])
+        i2Fi1 = np.array([[0, 1, 1], [1, 0, 0], [1, 0, 0]])
+        # i2Fi1 @ x1 = [4.5, 1.0, 1.0] and [3.0, -2.0, -2.0]. Norms^2 = 21.25 and 13.0
```

Comment:
could you upload a picture/plot?

Reply:
i would clarify that this norm of the normal, not of (a,b,c)

Reply:
better?